server/webapp/WEB-INF/rails.new/app/assets/javascripts/jquery.environmentVariableEdit.js

```diff
@@ -0,0 +1,65 @@
+(function ($) {
+  $.resetEnvironmentVariableOn = function (options) {
```

Comment:
I would recommend against writing global functions such as this one. Could you try encapsulating this in a wrapper class instead.